lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery/Random.hs

```diff
@@ -304,9 +304,9 @@ instance NFData (RndAnyState n p)
 -- | Initialize the HD random address discovery state from a root key and RNG
 -- seed.
 --
--- The type parameter is expected to be a ratio (between 0 and 100) of addresses
--- we ought to simply recognize as ours. So, giving @5 means that 5% of the
--- entire address space of the network will be considered ours, picked randomly.
+-- The type parameter is expected to be a ratio of addresses we ought to simply
+-- recognize as ours. It is expressed in tenth of percents, so "1" means 0.1%,
+-- "10" means 1% and 1000 means 100%.
```

Comment:
```suggestion
-- recognize as ours. It is expressed in tenths of a percent, so "1" means 0.1%,
-- "10" means 1%, and 1000 means 100%.
```

There is also [per mille](https://en.wikipedia.org/wiki/Per_mille), but it is rarely used in English, so I wouldn't recommend it. :smile_cat: